tests/baselines/reference/generatorTypeCheck39.errors.txt

```diff
@@ -13,7 +13,7 @@ tests/cases/conformance/es6/yieldExpressions/generatorTypeCheck39.ts(7,13): erro
 !!! error TS1163: A 'yield' expression is only allowed in a generator body.
         class C {
               ~

```

Comment:
It's kind of weird that we error on the node instead of the decorator itself. We should consider that in a separate change.